### PR TITLE
update: 1.16.2 + Version Depends

### PR DESCRIPTION
**Pull request by debouncedgit**
Depend on Minecraft Versions affected by the Issue, also make the mod website the Issue to see if it is fixed whenever it finally is.
Should work with the depended versions as far as I'm aware.

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -4,16 +4,20 @@
...
   "license": "WTFPL",
-  "environment": "*",
+  "environment": "client",
   "mixins": [
     "voyager.mixins.json"
   ],
   "depends": {
-    "fabricloader": ">=0.7.2"
+    "fabricloader": ">=0.7.2",
+    "minecraft": ["1.14", "1.14.1", "1.14.2", "1.14.3", "1.14.4", "1.15", "1.15.1", "1.15.2", "1.16", "1.16.1", "1.16.2"]
```
Comment (modmuss50) [inline src/main/resources/fabric.mod.json]: Just do `>=1.14`

File: src/main/resources/fabric.mod.json [hunk 2]
```diff
@@ -4,16 +4,20 @@
...
+  "description": "Take a voyage into modern java versions. (Fixes (MC-149777) a rare in StructureManager when using a Java Runtime Environment of 11 or above)",
   "authors": [
     "modmuss50"
   ],
+  "contact": {
+    "homepage": "https://bugs.mojang.com/browse/MC-149777"
+  },
   "license": "WTFPL",
-  "environment": "*",
+  "environment": "client",
```
Comment (modmuss50) [inline src/main/resources/fabric.mod.json]: This is not true, this also happens on servers.

**Review by modmuss50 — CHANGES_REQUESTED** [review]
Few changes required.

**Comment by debouncedgit** [comment]
Changes added, thanks!

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**